tests/integration-tests/tests/networking/test_security_groups.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+
+import boto3
+import pytest
+from assertpy import assert_that
+from cfn_stacks_factory import CfnStack
+from troposphere import Ref, Template
+from troposphere.ec2 import SecurityGroup, SecurityGroupIngress
+from utils import check_security_group, random_alphanumeric
+
+
+@pytest.mark.usefixtures("os", "scheduler", "instance")
+def test_additional_sg_and_ssh_from(region, custom_security_group, pcluster_config_reader, clusters_factory):
+    """
+    Test when additional_sg ssh_from are provided in the config file
+
+    The additional security group should be added to the head and compute nodes. The
+    """
+    custom_security_group_id = custom_security_group.cfn_resources["SecurityGroupResource"]
+    ssh_from = "10.11.12.0/32"
+    cluster_config = pcluster_config_reader(additional_sg=custom_security_group_id, ssh_from=ssh_from)
+    cluster = clusters_factory(cluster_config)
+    ec2_client = boto3.client("ec2", region_name=region)
+    instances = _get_instances_by_security_group(ec2_client, custom_security_group_id)
+    logging.info("Asserting that head node and compute node has the additional security group")
+    assert_that(len(instances)).is_equal_to(2)
```

Comment:
nit: you can use assert_that().is_length(2). Same everywhere else you assert the length